components/aws/sagemaker/README.md

```diff
@@ -6,7 +6,7 @@ With Amazon SageMaker Components for Kubeflow Pipelines (KFP), you can create an
 ## Components
 Amazon SageMaker Components for Kubeflow Pipelines offer an alternative to launching compute-intensive jobs in Kubernetes and integrate the orchestration benefits of Kubeflow Pipelines. The following Amazon SageMaker components have been created to integrate 6 key Amazon SageMaker features into your ML workflows. You can create a Kubeflow Pipeline built entirely using these components, or integrate individual components into your workflow as needed. 
 
-For an end-to-end tutorial using these components, see [Using Amazon SageMaker Components](https://sagemaker.readthedocs.io/en/stable/workflows/kubernetes/using_amazon_sagemaker_components.html).
+For an end-to-end tutorial using these components, see [Using Amazon SageMaker Components](https://docs.aws.amazon.com/sagemaker/latest/dg/kubernetes-sagemaker-components-for-kubeflow-pipelines.html).
```

Comment:
one more link on this page https://github.com/kubeflow/pipelines/blob/cc8e3fdec14047802a062f9608177c92c5f9dba2/samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md